cmd/cinder-csi-plugin/main.go

```diff
@@ -101,21 +101,21 @@ func handle() {
 	//Intiliaze mount
 	mount, err := mount.GetMountProvider()
 	if err != nil {
-		klog.V(3).Infof("Failed to GetMountProvider: %v", err)
+		klog.Infof("Failed to GetMountProvider: %v", err)
```

Comment:
All those logs will be printed when there is some error occurs, is it necessary to change to warning log level if the particular error doesn't break the process?

Reply:
agreed, here it is not required, it is sufficient to do it here only
https://github.com/kubernetes/cloud-provider-openstack/blob/master/cmd/cinder-csi-plugin/main.go#L120

Reply:
ok